project/FrgaalJavaCompiler.scala

```diff
@@ -220,7 +220,7 @@ object FrgaalJavaCompiler {
     withArgumentFile(allArguments) { argsFile =>
       val limitModulesArgs = Seq(
         "--limit-modules",
-        "java.base,jdk.zipfs,jdk.internal.vm.compiler.management,org.graalvm.locator,java.desktop,java.net.http"
+        "java.base,jdk.zipfs,jdk.internal.vm.compiler.management,java.desktop,java.net.http"
```

Comment:
GraalVM locator is needed to support the _"benchmark annotation processor"_ that generates JMH wrappers around Enso benchmarks.

Reply:
Updated the list of --limit-modules in c7820861264f7f02943f6c8edaf35e9701146aa0